koku/reporting/migrations/0207_savingsplan_postgres.py

```diff
@@ -0,0 +1,171 @@
+# Generated by Django 3.1.13 on 2021-11-15 16:29
```

Comment:
Please break this out into its own PR. That way, we can go ahead and apply it to the DB and have these new fields in place.

Reply:
Broke this into https://github.com/project-koku/koku/pull/3291 - once this is merged, we will update this PR.